configs/templates/ldap/ldap.yaml

```diff
@@ -3,14 +3,21 @@ apiVersion: v1
-  users: bGRhcC1hZG1pbjEsbGRhcC1hZG1pbjIsbGRhcC1hZG1pbjMsbGRhcC1hZG1pbjQsbGRhcC1hZG1pbjUsbGRhcC1hZG1pbjYsbGRhcC1hZG1pbjcsbGRhcC1hZG1pbjgsbGRhcC1hZG1pbjksbGRhcC1hZG1pbjEwLGxkYXAtYWRtaW4xMSxsZGFwLWFkbWluMTIsbGRhcC1hZG1pbjEzLGxkYXAtYWRtaW4xNCxsZGFwLWFkbWluMTUsbGRhcC1hZG1pbjE2LGxkYXAtYWRtaW4xNyxsZGFwLWFkbWluMTgsbGRhcC1hZG1pbjE5LGxkYXAtYWRtaW4yMCxsZGFwLXVzZXIxLGxkYXAtdXNlcjIsbGRhcC11c2VyMyxsZGFwLXVzZXI0LGxkYXAtdXNlcjUsbGRhcC11c2VyNixsZGFwLXVzZXI3LGxkYXAtdXNlcjgsbGRhcC11c2VyOSxsZGFwLXVzZXIxMCxsZGFwLXVzZXIxMSxsZGFwLXVzZXIxMixsZGFwLXVzZXIxMyxsZGFwLXVzZXIxNCxsZGFwLXVzZXIxNSxsZGFwLXVzZXIxNixsZGFwLXVzZXIxNyxsZGFwLXVzZXIxOCxsZGFwLXVzZXIxOSxsZGFwLXVzZXIyMCxsZGFwLW5vYWNjZXNzMSxsZGFwLW5vYWNjZXNzMixsZGFwLW5vYWNjZXNzMyxsZGFwLW5vYWNjZXNzNCxsZGFwLW5vYWNjZXNzNSxsZGFwLW5vYWNjZXNzNixsZGFwLW5vYWNjZXNzNyxsZGFwLW5vYWNjZXNzOCxsZGFwLW5vYWNjZXNzOSxsZGFwLW5vYWNjZXNzMTAsbGRhcC1ub2FjY2VzczExLGxkYXAtbm9hY2Nlc3MxMixsZGFwLW5vYWNjZXNzMTMsbGRhcC1ub2FjY2VzczE0LGxkYXAtbm9hY2Nlc3MxNSxsZGFwLW5vYWNjZXNzMTYsbGRhcC1ub2FjY2VzczE3LGxkYXAtbm9hY2Nlc3MxOCxsZGFwLW5vYWNjZXNzMTksbGRhcC1ub2FjY2VzczIwLGNvbGxpc2lvbi11c2VyLCBsZGFwLXNwZWNpYWw/LGxkYXAtc3BlY2lhbC4sbGRhcC1zcGVjaWFsXixsZGFwLXNwZWNpYWwkLGxkYXAtc3BlY2lhbCosbGRhcC1zcGVjaWFsKCxsZGFwLXNwZWNpYWwpLGxkYXAtc3BlY2lhbFssbGRhcC1zcGVjaWFsXSxsZGFwLXNwZWNpYWx7LGxkYXAtc3BlY2lhbH0sbGRhcC1zcGVjaWFsfCxsZGFwLXNwZWNpYWxA
+  #passwords: cmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMSxyaG9kc1BXIzEscmhvZHNQVyMxLHJob2RzUFcjMQ==
```

Comment:
These can be removed

Reply:
Sure @mrrajan, I'll remove the old usernames and password